packages/headless/src/controllers/search-parameter-manager/headless-search-parameter-manager.ssr.ts

```diff
@@ -10,6 +10,7 @@ export type {
   SearchParameterManagerInitialState,
   SearchParameterManagerState,
   SearchParameterManager,
+  SearchParameters,
```

Comment:
Won't re-exporting using `*` reduce maintenence overhead when new exports are added to the headless controllers?

Reply:
Good point. We already explicitly choose the types to export in the index file (just like we do with the search use case), so there's no need to selectively export stuff here. I'll correct that.